apps/crm/src/pages/api/auth/mailchimp/callback.ts

```diff
@@ -45,8 +47,8 @@ const handler: NextApiHandler = async (req, res) => {
   await mc.ping();
 
   return res.redirect(
-    `/configuration/mailchimp/oauth-success?token=${access_token}&email=${metadata.login.email}&dc=${metadata.dc}`
+    `/configuration/mailchimp/oauth-success?token=${access_token}&email=${metadata.login.email}&dc=${metadata.dc}`,
   ); // todo maybe move to cookie
 };
 
-export default handler;
+export default createProtectedHandler(handler, saleorApp.apl, ["MANAGE_APPS"]);
```

Comment:
though: maybe we should make MANAGE_APPS a default permission for `createProtectedHandler`? 🤔

Reply:
`createProtectedHandler` is from app-sdk, we should be careful with such opinions. I think its fine to be explicit since this is external library

Reply:
Right, we shouldn't do that on `app-sdk` level, but rather in apps monorepo - I'd like to prevent us from exposing unprotected endpoints on accident 😅